docker_kernel/magics/tag.py

```diff
@@ -35,24 +37,25 @@ def VALID_OPTIONS() -> dict[str, FlagDict]:
         }
     
     def _execute_magic(self) -> None:
-        target = self._args[0]
+        """
+        usage:
+        %tag source_image[:tag] target_image[:tag]
+        """
+        source_image = self._args[0]
+        target_image = self._args[1]
         try:
-            name, tag = target.split(":")
+            name, tag = target_image.split(":")
         except ValueError as e:
             # If no colon is provided the default tag is used
-            if ":" not in target:
-                name = target
-                tag = None
+            if ":" not in target_image:
+                name = target_image
+                tag = 'latest'
             else:
                 raise MagicError("Error parsing arguments:\n" + 
-                                f"\t\"{target}\" is not valid: invalid reference format")
+                                f"\t\"{source_image}\" is not valid: invalid reference format")
 
-        image_id = self._get_default_flag("image", "i", self._kernel._sha1)
-        if image_id is None:
+        if source_image is None:
             raise MagicError("No image specified")
 
-        self._kernel.tag_image(image_id, name, tag=tag)
-
-        image_str = image_id.removeprefix("sha256:")
-        image_str = f"{image_str[:10]}..." if len(image_str) >= 10 else image_str
-        self._kernel.send_response(f"Image {image_str} tagged")
+        self._kernel.tag_image(source_image, name, tag=tag)
+        self._kernel.send_response(f"Image {source_image} is tagged with: {name}:{tag}")
```

Comment:
Let's just move this response message into the kernel's `tag` method. So if we ever call `tag` from anywhere else the message will be the same.

Reply:
Done